functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -280,7 +278,12 @@ def _update_images(self, list_of_image_ids, new_image_tag_state, user_id, conn):
                     images_to_update = '{0}'.format(', '.join(image_ids_as_strings))
                     # TODO: find another way to do string subsitution that doesn't break this query
                     query = "UPDATE Image_Tagging_State SET TagStateId = {0}, ModifiedByUser = {2}, ModifiedDtim = now() WHERE ImageId IN ({1})"
-                    cursor.execute(query.format(new_image_tag_state,images_to_update,user_id))
+                    if not verify_old_tag_state:
+                        query = query.format(new_image_tag_state,images_to_update,user_id)
+                    else:
+                        inner_verify = "SELECT ImageId from Image_Tagging_State where TagStateId = {0} and ImageId IN ({1})".format(verify_old_tag_state, images_to_update)
```

Comment:
@samiyaakhtar I'm trying to think of a scenario where we would want to know if certain image ids are not being updated because they are in an expected state. Is there a way we can log this? I could see it being helpful in cases where one may have to debug something.

Reply:
Should we just be writing a log info message or adding them to some sort of a table for failed updates? It's probably too late for the latter, I can add some logging for sure. I'll just have to separate out the queries